src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -219,7 +219,8 @@ public function addFluentCommands(Grammar $grammar)
                 $value = $column->{$attributeName};
 
                 $this->addCommand(
-                    $commandName, compact('value', 'column')
+                    $commandName,
+                    compact('value', 'column')
```

Comment:
Thanks for the PR. Please revert the unrelated changes though, like this one, and the one at the bottom of the diff.

Reply:
:+1: Ready!